### PR TITLE
Landscape UI issues solved

### PR DESCRIPTION
**Pull request by codingblazer**
#275
The following issues have been resolved -
1. The avatar was not proper in some dpi screens - I have added scrollview for small screens like Nexus one and some rectification are made to make avatar properly work on different dpi screens.
2. Replay and continue buttons were placed below the question and thus couldn't fit in landscape mode. Now the button are placed to the right of question so it can fit in landscape. Thus, a new landscape xml is created - activity_scenario_over.xml
3. White box below the conversion has been removed by making listview as wrap_content.

Please review. :-)

### CODE REVIEW

**Comment by chhavip** [comment]
@codingblazer Please attach screenshots after the changes have been made.

**Comment by chhavip** [comment]
Code seems fine.

**Comment by codingblazer** [comment]
@chhavip Thanks for review. I am attaching the screenshots -

1. Before 
whatsapp image 2017-01-12 at 19 41 05
 After


2. Before 
whatsapp image 2017-01-12 at 19 41 05 1
After


3. Before
whatsapp image 2017-01-12 at 19 41 05 2
After

**Comment by codingblazer** [comment]
@chhavip I have resolved the merge conflict. Please review.

File: PowerUp/app/src/main/res/values/dimens.xml [hunk 1]
```diff
@@ -166,6 +165,8 @@
     <dimen name="button_layout_width">160dp</dimen>
     <dimen name="replay_layout_marginTop">365dp</dimen>
     <dimen name="margin_left_replay_button">30dp</dimen>
+    <dimen name="avatar_hair_marginTop">95dp</dimen>
+
```
Comment (chhavip) [inline PowerUp/app/src/main/res/values/dimens.xml]: remove extra lines

**Comment by codingblazer** [comment]
@chhavip Fixed.

**Comment by chhavip** [comment]
@codingblazer There seem to be missing drawables in this PR, have you added or removed any? I am facing the error of missing drawables for @drawable/replay_text and @drawable/endingscreen6. Please check for the same. @aanchal07 @anubhakushwaha , could you also test this PR on a device?

**Comment by codingblazer** [comment]
@chhavip No i didn't made any changes to drawables. I also checked the changes on github  and there seems no changed drawables. I will recheck and send PR if required :-)

**Comment by anubhakushwaha** [comment]
@codingblazer Your develop branch is not up-to-date with our develop branch and hence doesn't include the work of our latest merged PR due to which on testing we are  getting errors.Please update your remote repo and re-send the PR

**Comment by codingblazer** [comment]
@anubhakushwaha I have checked it but my branch is upto date with develop branch.

**Comment by codingblazer** [comment]
@anubhakushwaha @chhavip Problem solved. Can you check now. :-)

**Comment by anubhakushwaha** [comment]
@codingblazer I am not so convinced with the changes made to the activity_scenario_over.xml @chhavip @aanchal07 what are your opinions?

**Comment by codingblazer** [comment]
@anubhakushwaha Are you referring to code or the final look of UI ?

**Comment by anubhakushwaha** [comment]
@codingblazer final look of the UI.

**Comment by codingblazer** [comment]
@anubhakushwaha Can you suggest what should be done instead. I will make the changes. :-)

**Comment by anubhakushwaha** [comment]
@codingblazer In my opinion the unmodified work for activity_scenario_over.xml  seemed better else @chhavip @aanchal07 would suggest regarding the same.

**Comment by chhavip** [comment]
Seems fine to me, the scenario_over_activity can be fixed to resemble the original screen but not sure how much of that is required, opinions @aanchal07 ?

File: PowerUp/app/src/main/res/layout-land/activity_scenario_over.xml [hunk 2]
```diff
@@ -144,54 +146,53 @@
 
     <TextView
         android:id="@+id/scenarioTextView"
-        android:layout_width="@dimen/scenario_tv_width"
-        android:layout_height="@dimen/scenario_tv_height"
-        android:layout_marginTop="@dimen/scenario_description_margin_top"
+        android:layout_width="150dp"
+        android:layout_height="80dp"
+        android:layout_centerVertical="true"
```
Comment (anubhakushwaha) [inline PowerUp/app/src/main/res/layout-land/activity_scenario_over.xml]: @codingblazer any specific reasons to hardcode them?

**Comment by codingblazer** [comment]
@anubhakushwaha Made the changes. Please review :-)

**Comment by anubhakushwaha** [comment]
@codingblazer Please remove the conflicts.

**Comment by codingblazer** [comment]
@anubhakushwaha Resolved the conflicts. Please review :-)

**Comment by aanchal07** [comment]
@codingblazer Can you look into this issue? The text gets hidden by avatar head. I tested on Nexus 5 emulator.
image

**Comment by codingblazer** [comment]
@aanchal07 Yeah sure. Will do it.

**Comment by codingblazer** [comment]
@aanchal07 Solved the textview problem. Please review.

**Comment by aanchal07** [comment]
@codingblazer Great!
@chhavip @anubhakushwaha Can you review so we can merge it?

**Comment by anubhakushwaha** [comment]
@codingblazer Please retain the original work in activity_scenario_over.xml

**Comment by aanchal07** [comment]
Closing since conflicts are there.

**Pull request closed without merging**